README.md

```diff
@@ -14,9 +14,345 @@ ManageIQ plugin for the Hawkular provider.
 
 ## Development
 
-See the section on pluggable providers in the [ManageIQ Developer Setup](http://manageiq.org/docs/guides/developer_setup)
+This is a quick guide outlining all the necessary steps to get ManageIQ up and
```

Comment:
I have the idea that this doesn't fit well in the `README.md`. This is converting the readme file into a developer quick start, because readme doesn't have a lot of info. I think it's better to have this guide in its own file and place a link here.

Reply:
sure, we can move this to another file.

Reply:
I disagree, having ManageIQ working correctly is essential to test/developer on our provider code.

Reply:
I suppose they mean to have it more organized like this: https://github.com/ManageIQ/manageiq/blob/master/README.md